bazel/repository_locations.bzl

```diff
@@ -287,6 +287,28 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2020-11-23",
         cpe = "cpe:2.3:a:nghttp2:nghttp2:*",
     ),
+    com_github_nilslice_protolock = dict(
+        project_name = "protolock",
+        project_desc = "Track your .proto files and prevent changes to messages and services which impact API compatibility.",
+        project_url = "https://github.com/nilslice/protolock/",
+        version = "0.15.2",
+        sha256 = "26d0b491471866c4959a75a1418a577eab432782b9923fdf022535bb7c322374",
+        urls = ["https://github.com/nilslice/protolock/archive/refs/tags/v{version}.tar.gz"],
+        release_date = "2021-02-18",
+        use_category = ["api"],
+        strip_prefix = "protolock-{version}",
+    ),
+    com_github_emicklei_proto = dict(
+        project_name = "proto",
+        project_desc = "parser for Google ProtocolBuffers definition",
+        project_url = "https://github.com/emicklei/proto/",
+        version = "1.7.0",
+        sha256 = "e93272fea9e4f993b9d160440bf980d015970147907090834492771bb1c4510c",
+        urls = ["https://github.com/emicklei/proto/archive/refs/tags/v{version}.tar.gz"],
+        release_date = "2019-10-05",
+        use_category = ["api"],
+        strip_prefix = "proto-{version}",
+    ),
```

Comment:
scorecard --repo=https://github.com/emicklei/proto
```
RESULTS
-------
Repo: github.com/emicklei/proto
|--------|------------|-----------------------------|
| STATUS | CONFIDENCE |            NAME             |
|--------|------------|-----------------------------|
| Pass   |         10 | Binary-Artifacts            |
|--------|------------|-----------------------------|
| Pass   |         10 | CI-Tests                    |
|--------|------------|-----------------------------|
| Pass   |         10 | Code-Review                 |
|--------|------------|-----------------------------|
| Pass   |         10 | Contributors                |
|--------|------------|-----------------------------|
| Pass   |         10 | Frozen-Deps                 |
|--------|------------|-----------------------------|
| Pass   |         10 | Token-Permissions           |
|--------|------------|-----------------------------|
| Pass   |         10 | Vulnerabilities             |
|--------|------------|-----------------------------|
| Fail   |         10 | Active                      |
|--------|------------|-----------------------------|
| Fail   |          3 | Automatic-Dependency-Update |
|--------|------------|-----------------------------|
| Fail   |         10 | Branch-Protection           |
|--------|------------|-----------------------------|
| Fail   |         10 | CII-Best-Practices          |
|--------|------------|-----------------------------|
| Fail   |         10 | Fuzzing                     |
|--------|------------|-----------------------------|
| Fail   |          0 | Packaging                   |
|--------|------------|-----------------------------|
| Fail   |          3 | Pull-Requests               |
|--------|------------|-----------------------------|
| Fail   |         10 | SAST                        |
|--------|------------|-----------------------------|
| Fail   |         10 | Security-Policy             |
|--------|------------|-----------------------------|
| Fail   |          0 | Signed-Releases             |
|--------|------------|-----------------------------|
| Fail   |         10 | Signed-Tags                 |
|--------|------------|-----------------------------|
```

Reply:
If we were to fork Protolock, I'd ask "why can't we rely on the standard protoc plugin `FileDescriptorProto` for the AST parse tree here?".